src/main/java/bio/terra/cli/app/utils/VersionCheckUtils.java

```diff
@@ -40,8 +41,8 @@ public static boolean isOlder(
       return false;
     }
 
-    var currentVersion = Version.parse(currentVersionString);
-    var oldestSupportedVersion = Version.parse(oldestSupportedVersionString);
+    ModuleDescriptor.Version currentVersion = Version.parse(currentVersionString);
```

Comment:
In general, I don't think we need to get rid of `var` when it's clear what the object type is e.g. `Version.parse` returning a `Version` object

Reply:
noted